tasks/task457_matres_conditional_classification.json

```diff
@@ -11,18 +11,23 @@
     "Domains": [
         "News"
     ],
-    "Definition": "You will be given a context and a verb separated with a \n, and you have to answer if the given verb implies a hypothetical, or condition or not. We say a verb is hypothetical if it involves or is based on a suggested idea or theory. A verb is conditional if something is essential to the appearance or occurrence of something else. The output should be \"Yes\" if the verb is conditional or hypothetical and \"No\" otherwise.",
+    "Definition": "You will be given a context and a verb separated with a newline character, and you have to answer if the given verb implies a hypothetical, or condition or not. We say a verb is hypothetical if it involves or is based on a suggested idea or theory. A verb is conditional if it is in conditional sentences and something is essential to the appearance or occurrence of something else. Note that an event that happened in the past or it is happening now cannot be conditional. The output should be \"Yes\" if the verb is conditional or hypothetical and \"No\" otherwise.",
```

Comment:
This definition can be worded better. For example:
`You will be given a context and a verb separated with a newline character. You have to decide if the given verb implies a hypothetical or conditioned action or not. We say a verb is hypothetical if it involves or is based on a suggested idea or theory. A verb is conditional if it is in a conditional sentence, and its occurrence relies on something else. Note that an event that happened in the past or is happening now cannot be conditional. The output should be \"Yes\" if the verb is conditional or hypothetical and \"No\" otherwise.`